### PR TITLE
HTML5 - ActionBar Button Positioning

### PR DESCRIPTION
**Pull request by KDSBrowne**
Positions the actions bar join audio, status and mute buttons to the center of the screen. 
These buttons stay centered if the user has not joined audio, joins audio or is made presenter.

Also centers the slide controls for the presenter.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss [hunk 1]
```diff
@@ -23,10 +27,30 @@
...
+.center,
+.nonVoicePresenter,
+.voicePresenter,
+.nonVoiceUser,
+.voiceUser {
   flex: 1;
 }
 
+.nonVoicePresenter {
+  margin-left: -11.6em;
```
Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss]: What are all those magic numbers?

File: bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx [hunk 2]
```diff
@@ -43,10 +45,12 @@ export default class ActionsBar extends Component {
     );
   }
 
-  renderForUser() {
+  renderForUser(isVoiceUser) {
```
Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx]: I know this was not introduced in this PR but I couldn't help but notice that `renderForUser` and `renderForPresenter` contain almost the same code except for some lines (a div and it's wrapped componend and the style classes). They could be consolidated into only one method.

File: bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss (hunk 1 above)
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss]: they are the values used to center the div containing the buttons on the action bar

Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss]: and what happen if a new button is added? Or the button size change? We should avoid fixed values, adding a new button shouldn't break the layout for example.

Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss]: You should center things using flex-box properties and not margin. Things should be centered not mattering its size or the parent size. Margin should be used to add margin and not to center things, there is better alternatives to it.

File: bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx (hunk 2 above)
Comment (perroned) [inline bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx]: i assume separating the renders was just for future proofing so if additional controls are added or different views then whoever implements it won't have to split it. The 2 render methods could remain the same but perhaps the shared code could be re used?

**Pull request closed without merging**